libdrgn/dwarf_info.c

```diff
@@ -5336,6 +5336,77 @@ drgn_object_from_dwarf(struct drgn_debug_info *dbinfo,
 					       function_die, regs, ret);
 }
 
+DEFINE_VECTOR(drgn_char_p_vector, const char *);
+
+static struct drgn_error *add_dwarf_enumerators(Dwarf_Die *enumeration_type,
+						struct drgn_char_p_vector *vec)
+{
+	Dwarf_Die child;
+	int r = dwarf_child(enumeration_type, &child);
+	while (r == 0) {
+		if (dwarf_tag(&child) == DW_TAG_enumerator) {
+			const char *name = dwarf_diename(&child);
```

Comment:
We need to check that `name` isn't `NULL` (which I realize is also a bug in `find_dwarf_enumerator()`).

Reply:
Fixed the `find_dwarf_enumerator()` version of this bug in b3a5051ff446a6e54aa217dda87685bd4a494f9c.